mintapi/api.py

```diff
@@ -1,5 +1,6 @@
 from datetime import date, datetime
 from dateutil.relativedelta import relativedelta
+import mintapi.constants as constants
```

Comment:
Normally I would do `from mintapi import constants` but this does make the imports a little visually cleaner to do it this way. Is there any difference?

Reply:
No difference. I'd say just consistency. I think I veered off of convention so I can change it.